### PR TITLE
Fuzzing: Add inject.IntoResourceFile fuzzer

### PR DESCRIPTION
**Pull request by AdamKorcz**
Adds fuzzer that targets inject.IntoResourceFile.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[x] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[x] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[x] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: tests/fuzz/oss_fuzz_build.sh [hunk 1]
```diff
@@ -21,7 +21,12 @@ compile_go_fuzzer istio.io/istio/tests/fuzz FuzzParseMeshNetworks fuzz_parse_mes
 compile_go_fuzzer istio.io/istio/tests/fuzz FuzzValidateMeshConfig fuzz_validate_mesh_config
 compile_go_fuzzer istio.io/istio/tests/fuzz FuzzInitContext fuzz_init_context
 compile_go_fuzzer istio.io/istio/tests/fuzz FuzzAnalyzer fuzz_analyzer
+compile_go_fuzzer istio.io/istio/tests/fuzz FuzzIntoResourceFile fuzz_into_resource_file
 
 
 # Create seed corpora:
 zip "${OUT}"/fuzz_analyzer_seed_corpus.zip "${SRC}"/istio/galley/pkg/config/analysis/analyzers/testdata/*.yaml
+zip "${OUT}"/fuzz_into_resource_file_seed_corpus.zip ./pkg/kube/inject/testdata/inject/*.yaml.injected
```
Comment (howardjohn) [inline tests/fuzz/oss_fuzz_build.sh:29]: the *.injected ones are the output, not the inputs. I think we want the .yaml without .injected suffix

Comment (AdamKorcz) [reply tests/fuzz/oss_fuzz_build.sh]: Updated

**Review by howardjohn — APPROVED** [review]

**Pull request merged**